lib/src/css_mirroring/directives_processor.dart

```diff
@@ -0,0 +1,45 @@
+part of scissors.src.css_mirroring.bidi_css_generator;
+
+class DirectionIndependentDirectivesProcessor extends NodeProcessor {
+  DirectionIndependentDirectivesProcessor();
+
+  process(EditConfiguration _editConfig, TopLevelsPair _topLevelsPair,
+      int _iTopLevel) {
```

Comment:
those underscores just add noise throughout the file: they're meant for private variables, not locals / params.